### ISSUE TITLE
About the rich feedback model release

### ISSUE THREAD

**Issue by srymaker** [google-research-datasets/richhf-18k#3]
Thanks for your great work!
Will the rich feedback model be released?I'd love to test and experience the model and apply it to my own tasks！

**Comment by leebird** [comment]
Hello, currently we don't have the plan to release the model.

**Comment by densechen** [comment]
@leebird Looking forward to the rich feedback model...

**Comment by ghost** [comment]
Looking forward

**Comment by leebird** [comment]
Thanks for all the interests in our work! Due to company policies (related to productization etc.) we could not open source the model. We have included details of how to reproduce the results in our paper. If you have further questions please email the corresponding authors, and we'd be happy to help you reproduce the results.

**Comment by srymaker** [comment]
Hello, can you tell me how you trained the reward model, like which layers were frozen and which tuning method was used?

**Comment by leebird** [comment]
Hi @srymaker , we finetuned all the layers in the model, including the ViT component. We tried freezing the ViT component but it didn't work well, especially for the heatmap tasks. Experiment details including hyperparameters and optimizer can be found in Section 9 in the paper.

**Comment by srymaker** [comment]
Thank you for your answer. Do all layers refer to the encoder and decoder in t5?

**Comment by leebird** [comment]
@srymaker yes, all the layers are from the ViT and T5 encoder/decoder. Note there is a pretraining stage for the ViT and T5 layers on multimodal data as they were originally pretrained on unimodal data only.

**Comment by Nieleilei** [comment]
> @srymaker yes, all the layers are from the ViT and T5 encoder/decoder. Note there is a pretraining stage for the ViT and T5 layers on multimodal data as they were originally pretrained on unimodal data only.


hello，Besides WebLI, what other pre-training datasets are available here?

**Comment by Nieleilei** [comment]
> @srymaker yes, all the layers are from the ViT and T5 encoder/decoder. Note there is a pretraining stage for the ViT and T5 layers on multimodal data as they were originally pretrained on unimodal data only.

For single-modal pretraining tasks, are only the natural image captioning tasks on the WebLI dataset used? What other tasks are included?

**Comment by Nieleilei** [comment]
> @srymaker yes, all the layers are from the ViT and T5 encoder/decoder. Note there is a pretraining stage for the ViT and T5 layers on multimodal data as they were originally pretrained on unimodal data only.

Hello, For single-modal pretraining tasks, are only the natural image captioning tasks on the WebLI dataset used? What other tasks are included?
